google/cloud/aiplatform/constants/prediction.py

```diff
@@ -0,0 +1,138 @@
+    "us-docker.pkg.dev/vertex-ai/prediction/sklearn-cpu.0-20:latest",
+    "europe-docker.pkg.dev/vertex-ai/prediction/sklearn-cpu.0-20:latest",
+    "asia-docker.pkg.dev/vertex-ai/prediction/sklearn-cpu.0-20:latest",
+]
+
+TF_CONTAINER_URIS = [
+    "us-docker.pkg.dev/vertex-ai/prediction/tf2-cpu.2-6:latest",
+    "europe-docker.pkg.dev/vertex-ai/prediction/tf2-cpu.2-6:latest",
+    "asia-docker.pkg.dev/vertex-ai/prediction/tf2-cpu.2-6:latest",
+    "us-docker.pkg.dev/vertex-ai/prediction/tf2-gpu.2-6:latest",
+    "europe-docker.pkg.dev/vertex-ai/prediction/tf2-gpu.2-6:latest",
+    "asia-docker.pkg.dev/vertex-ai/prediction/tf2-gpu.2-6:latest",
+    "us-docker.pkg.dev/vertex-ai/prediction/tf2-cpu.2-5:latest",
+    "europe-docker.pkg.dev/vertex-ai/prediction/tf2-cpu.2-5:latest",
+    "asia-docker.pkg.dev/vertex-ai/prediction/tf2-cpu.2-5:latest",
+    "us-docker.pkg.dev/vertex-ai/prediction/tf2-gpu.2-5:latest",
+    "europe-docker.pkg.dev/vertex-ai/prediction/tf2-gpu.2-5:latest",
+    "asia-docker.pkg.dev/vertex-ai/prediction/tf2-gpu.2-5:latest",
+    "us-docker.pkg.dev/vertex-ai/prediction/tf2-cpu.2-4:latest",
+    "europe-docker.pkg.dev/vertex-ai/prediction/tf2-cpu.2-4:latest",
+    "asia-docker.pkg.dev/vertex-ai/prediction/tf2-cpu.2-4:latest",
+    "us-docker.pkg.dev/vertex-ai/prediction/tf2-gpu.2-4:latest",
+    "europe-docker.pkg.dev/vertex-ai/prediction/tf2-gpu.2-4:latest",
+    "asia-docker.pkg.dev/vertex-ai/prediction/tf2-gpu.2-4:latest",
+    "us-docker.pkg.dev/vertex-ai/prediction/tf2-cpu.2-3:latest",
+    "europe-docker.pkg.dev/vertex-ai/prediction/tf2-cpu.2-3:latest",
+    "asia-docker.pkg.dev/vertex-ai/prediction/tf2-cpu.2-3:latest",
+    "us-docker.pkg.dev/vertex-ai/prediction/tf2-gpu.2-3:latest",
+    "europe-docker.pkg.dev/vertex-ai/prediction/tf2-gpu.2-3:latest",
+    "asia-docker.pkg.dev/vertex-ai/prediction/tf2-gpu.2-3:latest",
+    "us-docker.pkg.dev/vertex-ai/prediction/tf2-cpu.2-2:latest",
+    "europe-docker.pkg.dev/vertex-ai/prediction/tf2-cpu.2-2:latest",
+    "asia-docker.pkg.dev/vertex-ai/prediction/tf2-cpu.2-2:latest",
+    "us-docker.pkg.dev/vertex-ai/prediction/tf2-gpu.2-2:latest",
+    "europe-docker.pkg.dev/vertex-ai/prediction/tf2-gpu.2-2:latest",
+    "asia-docker.pkg.dev/vertex-ai/prediction/tf2-gpu.2-2:latest",
+    "us-docker.pkg.dev/vertex-ai/prediction/tf2-cpu.2-1:latest",
+    "europe-docker.pkg.dev/vertex-ai/prediction/tf2-cpu.2-1:latest",
+    "asia-docker.pkg.dev/vertex-ai/prediction/tf2-cpu.2-1:latest",
+    "us-docker.pkg.dev/vertex-ai/prediction/tf-cpu.1-15:latest",
+    "europe-docker.pkg.dev/vertex-ai/prediction/tf-cpu.1-15:latest",
+    "asia-docker.pkg.dev/vertex-ai/prediction/tf-cpu.1-15:latest",
+    "us-docker.pkg.dev/vertex-ai/prediction/tf-gpu.1-15:latest",
+    "europe-docker.pkg.dev/vertex-ai/prediction/tf-gpu.1-15:latest",
+    "asia-docker.pkg.dev/vertex-ai/prediction/tf-gpu.1-15:latest",
+]
+
+# Pattern:
+# {user_provided_framework: framework_name_in_uri}
+_FRAMEWORK_TO_URI_REF = {
```

Comment:
I like the UX of this approach but it may be less ambiguous to provide a values, based on the container uri framework keyword.